app/routes.js

```diff
@@ -1,9 +1,17 @@
-// const controller = require('./controllers/controller');
+// Controllers
 const { healthCheck } = require('./controllers/healthCheck');
+const userController = require('./controllers/users');
+
+// Check Schemas
+const { createUserValidator } = require('./middlewares/schemas/users');
+const { validationsFields } = require('./middlewares/validationFields');
+const { validateUserByEmail } = require('./middlewares/database/users');
 
 exports.init = app => {
   app.get('/health', healthCheck);
-  // app.get('/endpoint/get/path', [], controller.methodGET);
-  // app.put('/endpoint/put/path', [], controller.methodPUT);
-  // app.post('/endpoint/post/path', [], controller.methodPOST);
+  app.post(
+    '/users',
+    [createUserValidator, validationsFields, validateUserByEmail],
```

Comment:
Maybe find a way to group the first two middlewares to avoid having to always put two when checking schemas. If you want to, I can give you a hint

Reply:
Sure, tell me.